Mage/Logger/Logger.cs

```diff
@@ -4,39 +4,47 @@
 using Wizcorp.MageSDK.Event;
 using Wizcorp.MageSDK.Log.Writers;
 
-namespace Wizcorp.MageSDK.Log {
-	public class Logger {
+namespace Wizcorp.MageSDK.Log
+{
+	public class Logger
```

Comment:
https://github.com/aspect-kdms/kdms-client-aspect/tree/develop/Assets/External/WizcorpUtils/Logger

This one